static-assets/components/cstudio-forms/forms-engine.js

```diff
@@ -1680,21 +1682,26 @@ var CStudioForms =
                 .subscribe(({ allowed, modifiedValue, target }) => {
                   if (allowed) {
                     if (modifiedValue) {
-                      CStudioAuthoring.Utils.showConfirmDialog(
-                        null,
-                        formatMessage(formEngineMessages.createPolicy, { originalPath: entityId, path: modifiedValue }),
-                        () => {
+                      CStudioAuthoring.Utils.showConfirmDialog({
+                        body: formatMessage(formEngineMessages.createPolicy, {
+                          originalPath: entityId,
+                          path: modifiedValue
+                        }),
+                        onOk: () => {
                           saveContent();
+                        },
+                        onCancel: () => {
+                          setButtonsEnabled(true);
```

Comment:
Where are these getting disabled?

Reply:
They're getting disabled at the beginning of the save function (right when clicking on save button)